src/superphot_plus/fit_numpyro.py

```diff
@@ -88,6 +89,8 @@ def run_mcmc(lc, sampler="NUTS", t0_lim=None, plot=False):
     plot : bool, optional
         If True, associated plots will be generated and saved. Defaults
         to False.
+    telescope : Telescope
+        Information about telescope used to collect LC data.
```

Comment:
The `telescope` parameter in the comment does not match the new parameter (`priors`).